app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -298,6 +285,23 @@ class SuggestedEditsTasksFragment : Fragment() {
         tasksContainer.visibility = VISIBLE
     }
 
+    private fun setUserStatsTooltips() {
```

Comment:
@cooltey  This function does much more than set up tooltips.  Can we give it a more meaningful name or break it up?

Reply:
Sure! Will do it.